clients/hadoopfs/src/main/java/io/lakefs/Constants.java

```diff
@@ -9,6 +9,8 @@ public class Constants {
     public static final String SECRET_KEY_KEY_SUFFIX = "secret.key";
     public static final String ENDPOINT_KEY_SUFFIX = "endpoint";
     public static final String LIST_AMOUNT_KEY_SUFFIX = "list.amount";
+    public static final String OC_DELETE_JOB_BRANCH = "committer.delete.job_branch";
```

Comment:
These configurations can both pass as a branch name configuration.
Would go with something as `committer.cleanup.delete_job_branch` - to clarify it's a boolean.

Reply:
We ended up with these names:

* `spark.hadoop.fs.lakefs.committer.delete_job_branch`
* `spark.hadoop.fs.lakefs.committer.delete_task_branch`